src/manager/msWindowManager.js

```diff
@@ -362,7 +368,7 @@ var MsWindowManager = class MsWindowManager extends MsManager {
         let workspaceIndex = Me.msWorkspaceManager.primaryMsWorkspaces.indexOf(
             msWindow.msWorkspace
         );
-        msWindow.app.launch(0, workspaceIndex, false);
+        msWindow.app.open_new_window(workspaceIndex);
```

Comment:
I think I prefer keep the launch method it's seem to do better job to launch a app in the right workspace

Reply:
But open_new_window is much faster if the app is already running, and if it is now then it would probably result in launh anyway... Also, it does not allow using apps with accelerated graphics (if the app was launched in this mode and is still running).

Reply:
Let's keep launch and add a new hotkey `super+enter` to open a new window of the current focused window application